x-pack/plugin/slm/src/test/java/org/elasticsearch/xpack/slm/SnapshotLifecyclePolicyTests.java

```diff
@@ -401,15 +475,16 @@ protected SnapshotLifecyclePolicy createTestInstance() {
 
     @Override
     protected SnapshotLifecyclePolicy mutateInstance(SnapshotLifecyclePolicy instance) {
-        switch (between(0, 5)) {
+        switch (between(0, 6)) {
```

Comment:
This is completely optional, but I usually like to refactor these methods to something like:
```java
var id = instance.getId();
var name = instance.getName();
...
switch (between(0, 6)) {
  case 0: id = id + randomAlphaOfLength(2);
  case 1: name = name + randomAlphaOfLength(2);
  ...
}
return new SnapshotLifecyclePolicy(
  id,
  name,
  ...
);
```
That significantly reduces the size of the method and thus makes it a little bit easier to read (IMO). Again, not mandatory, so it's up to you if you feel like doing that.